### ISSUE TITLE
You can create Roku-Channel

### ISSUE THREAD

**Issue by xoceunder** [BYUI-Web/BYU-I-Roku-Channel#3]
You can create this Roku-Channel app

**Comment by xoceunder** [comment]
@jamesblack

**Comment by SamJBarney** [comment]
@xoceunder are you part of BYU-Idaho?
Whether you are or not, this channel is being created by the BYU-Idaho University Relations team and is currently awaiting approval for deployment. So having @jamesblack "create" is unecessary, especially since he is not authorized to do so.

**Comment by jshcrowthe** [comment]
@SamJBarney Regardless of who "owns" the rights to this channel the code is living as a public repo on github. Meaning at the moment **ANYONE** who would like to come along and use this code to make a Roku channel (including using the exact same codebase) is able to without permission from anyone.

I would recommend that you work with @JeffreyDunster on getting this transferred to a private repo (if you are thinking that this will take a while). Also you might consider creating a LICENSE for this repo outlining its usage (so you don't have people publishing this code which they legally can atm).

**Comment by SamJBarney** [comment]
Technically, until a license is applied, it is considered all rights reserved.

**Comment by jamesblack** [comment]
I would like to just quickly chime in: 

First I have no idea whats going on, as I do not wish to make a roku channel, that being said. @SamJBarney the current copyright owner of this code is @jshcrowthe unless he had signed a contract stating that all work he did would be assigned to Byu Idaho. You are correct, the copyright default applies when there is no license file. So it is currently All Rights Reserved. But those reserved rights are to whoever wrote it unless otherwise stated in a contract.

I don't know is @jshcrowthe signed such a contract, but if he didn't then technically Byu Idaho would be unable to publish this code without his express permission, nor would they be able to claim said code.

PS. I would have turned down the job, but a fun fact is that I am "authorized" to do many things, that could involved a BYU-I channel, try not to use "authorization" as a point of attack on a first swing. It upsets people.

**Comment by SamJBarney** [comment]
I apologize if I upset you, as it was not my intention.

**Comment by SamJBarney** [comment]
Out of curiosity @jamesblack , if he wrote it on BYU-Idaho's time, on behalf of BYU-Idaho, does he still have license for it?

**Comment by jshcrowthe** [comment]
@JeffreyDunster and have talked and will work on resolving this. Thanks all!

**Comment by JeffreyDunster** [comment]
BYU-Idaho employed Josh to create this channel for the university's A/V content. The university owns the copyright on this project. We used open-source code from other projects and intended for this channel to also be open-source. However, the A/V assets, feeds and configuration files for the channel are protected by university copyright. We have neglected to apply a formal license to this code base and I'm reviewing the matter with the university's intellectual property office. BYU-Idaho does not have an established history with open-source, or creative commons licenses. I'm happy to be breaking ground here!

As for publishing the Roku channel, it's under consideration by University Relations. The Roku is a new publishing medium that needs formal approval before we can release and support this channel. This project was a proof of concept that I hope gets to see the light of day!
